k8s/base/app/invoice-cronjob.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: invoice-cronjob
+spec:
+  schedule: "0 13 1 * *" # TODO Is this the correct schedule?
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: invoice-cronjob
+            image: ghcr.io/nerc-project/coldfront-nerc:main
```

Comment:
I wouldn't use main, put a patch with the overlay for `prod` where you specify the same version.